Kudu.Core/Infrastructure/PathUtils/PathWindowsUtility.cs

```diff
@@ -108,6 +108,11 @@ internal override string ResolveNpmJsPath()
 
         internal override string ResolveMSBuild15Dir()
         {
+            if (ScmHostingConfigurations.UseLatestMSBuildInsteadOfMSBuild15)
+            {
+                return Path.Combine(ResolveLatestMSBuildDir());
```

Comment:
Let say we ship new MSBuild tomorrow, will ResolveLatestMSBuildDir return that new MSBuild version?   If so, it means that every time we ship new MSBuild, we risk of breaking many apps, correct?   To avoid this, we should nominate one MSBuild (say latest one as off today) to be used instead of MSBuild 14 and 15.    Shipping new MSBuild would not impact that.  Thought?

Reply:
That's a good point. 

I agree it will increase the risk of breaking apps because of compatibility issues or something like that. However by pinning to one version, we will tie our hands to maintaining that particular version while adding new MSBuild versions as new dotnet versions will require new MSBuild versions. 

Since MSBuild size keeps increasing, it would contribute to RU size as well. 

Thoughts?

Reply:
Updated the PR and pointed to latest version of MSBuild 16.